packages/components/modal/src/ModalConfirm/ModalConfirm.tsx

```diff
@@ -47,7 +47,12 @@ export interface ModalConfirmProps {
   /**
    * The intent of the ModalConfirm. Used for the secondary Button.
    */
-  secondaryIntent?: 'primary' | 'positive' | 'negative' | 'transparent';
+  secondaryIntent?:
+    | 'primary'
+    | 'positive'
+    | 'negative'
+    | 'transparent'
+    | 'secondary';
```

Comment:
I thought, based on the designs, that the secondary button would always be `secondary `